utilities/examine_focus_w_grid.py

```diff
@@ -72,33 +76,28 @@ def variance_of_laplacian(image):
 mv_avg_freq = 5  # Number of frames to perform moving average on
 laplace_array = np.full((num_rows, num_cols, mv_avg_freq), None)
 
-# Initialize camera and grab a reference to the raw camera capture
-camera = PiCamera()
-camera.rotation = 180
-camera.resolution = (640, 480)
-camera.framerate = 30
-rawCapture = PiRGBArray(camera, size=(640, 480))
+# Open a camera for video capturing
+cap = cv2.VideoCapture(0)
 
-# Allow the camera to warmup
-time.sleep(0.1)
 
-# Capture frames from the camera
-for frame in camera.capture_continuous(rawCapture,
-                                       format='rgb',
-                                       use_video_port=True):
-    # Grab the raw NumPy array representing the image
-    color_image = frame.array
+def show_frame():
+    global tkinter_image, mv_avg_count
 
-    tkinter_image = ImageTk.PhotoImage(image=Image.fromarray(color_image))
-    canvas.itemconfig(image, image=tkinter_image)
+    ret, frame = cap.read()
+    frame = cv2.resize(frame, (display_width, display_height))
+    frame = cv2.rotate(frame, cv2.ROTATE_180)
+    color_image = cv2.cvtColor(frame, cv2.COLOR_BGR2RGBA)
+    gray_image = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
 
-    gray_image = cv2.cvtColor(color_image, cv2.COLOR_RGB2GRAY)
+    image_data = Image.fromarray(color_image)
+    tkinter_image = ImageTk.PhotoImage(image=image_data)
+    canvas.itemconfig(image_on_canvas, image=tkinter_image)
 
-    for i in range(num_cols):
-        for j in range(num_rows):
-            left = int(sector_width * i)
+    for i in range(num_rows):
+        for j in range(num_cols):
+            left = int(sector_width * j) + 1
```

Comment:
This is where we are dealing with the issue of the line of white pixels on the left of the image, right? (If yes, I am guessing it turned out it was only 1 pixel wide?) Also, if yes, a later maintainer/developer might wonder why there is this mysterious ```+ 1``` so I think this warrants an explanatory comment so it's clear what is going on.

Also: am I right in thinking that means we will crop off the left-most line of pixels in every sector (i.e. not just the ones against the left side of the image)?  Doesn't seem too big a deal to do that for sake of making this solution easy--doesn't feel like it is going to affect the utility of the tool except in infinitesimally rare corner cases--but calling it out nonetheless.

Reply:
Indeed, this is where we are dealing with the issue of the line. I have yet to figure out what exactly that line looks like, so I cannot say that it's a line of white pixels. I'm working on adding a condition so that only the left column gets 1px added on the left side. However, I'm waiting to hear back about the traversal of the 2D array since the implementation depends on the traversal order. I'll definitely make sure to add an explanatory comment.

Reply:
Sounds good.